lib/queueservice.js

```diff
@@ -107,12 +107,10 @@ export default class TaskQueue {
     let claim;
     try {
       claim = await this.stats.timeGen(
-        'tasks.time.claim',
+        'claimTask',
```

Comment:
I don't like this... Let's do that with the client API stats handler... See two PRs filed against you :)

IMO if we want to record statistics about a method I see three options:
 1. Add stats recording (or callback/event) from with method
 2. Monkey patch the method to record stats
 3. Wrap the entire thing the method lives on and add stats recording there

Adding stats at call-sites throughout the code is bad. It reduces readability and adds things we don't want to see when we review logic. It's also error prone, let's strive to only use it in corner cases (yeah, we have a lot of those :))